src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -130,24 +130,28 @@ bool convert_precision(ov::pass::PassBase& pass,
     };
 
     auto convert_node_output_precision = [&](const std::shared_ptr<ngraph::Node>& node) {
+        // Handle case with Constants as they can have consumers from other nGraph Function object
+        const auto it = const_to_internal_output.find(node.get());
+        if (it != const_to_internal_output.end()) {
+            return fuse_type_to_constant(node, to, it->second);
+        }
+        // Check that node type exists in map and we can fuse type into node
+        auto t2f_it = type_to_fuse.find(node->get_type_info());
+        if (t2f_it == type_to_fuse.end()) {
+            return false;
+        }
+        bool result = false;
```

Comment:
The intention of changes in `convert_node_output_precision` is to address https://github.com/openvinotoolkit/openvino/pull/14229#discussion_r1032082670 and return `true` from `fuse_type_to_unique_v10` after precision change. 

For verification, the following function should be updated as well:
https://github.com/openvinotoolkit/openvino/blob/b475225a5436074d17b24e329f1b80bd442399dd/src/common/transformations/src/transformations/convert_precision.cpp#L345-L352

Reply:
Sorry, but looks like this comment is not applied.

Reply:
Right, those changes were removed during a commit revert. I'll add it back.